lib/onegini.dart

```diff
@@ -57,8 +57,11 @@ class Onegini {
       });
       eventListener.listen();
       return removedUserProfileListFromJson(removedUserProfiles);
-    } on PlatformException catch (error) {
-      throw error;
+    } on TypeError catch (error) {

```

Comment:
Won't that catch **only** `TypeError`? What about handling different errors that can occur? I think we could use dart `on` keyword, like in this example:

```
try {
  // ...
} on SomeException catch(e) {
 //Handle exception of type SomeException
} catch(e) {
 //Handle all other exceptions
}
```

Reply:
As discussed, we will let all other errors bubble up as they will just reject the future, the develop can then handle these as they want.